setup.py

```diff
@@ -85,7 +85,7 @@ def is_platform_mac():
 try:
     if not _CYTHON_INSTALLED:
         raise ImportError('No supported version of Cython installed.')
-    from Cython.Distutils import build_ext as _build_ext
```

Comment:
well that will fail on older versions of cython wouldn't it?

need to check in a try except 

further we use older (and newer versions) of cython in various builds